Sources/SwiftRepo/Repository/StoreModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  StoreModel.swift
+//  SwiftRepo
+//
+//  Created by Carter Foughty on 10/1/24.
+//
+
+import Foundation
+
+/// A model that can be placed and fetched in a store.
```

Comment:
I think it would be helpful to talk about the fact that this interface is intended for models that will be retrieved by the app through database query rather than published by a repository.